raiden-dapp/src/components/AmountDisplay.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div
+    @mouseover="exactAmount ? (displayExactAmount = true) : null"
+    @mouseleave="exactAmount ? (displayExactAmount = false) : null"
+  >
+    <span v-if="displayExactAmount">{{ amount }} {{ symbol }}</span>
+    <span v-else>
+      {{ amount | displayFormat(amount.decimals) }} {{ symbol }}
```

Comment:
if `amount` is a string this would probably fail because it doesn't have `decimals`

Reply:
Yes :) I will pass a token instead.